networks/cardano/tokens/e2087044f1a5028aec705eb05609370754d64587adbdc27eeb33005854414e47.json

```diff
@@ -0,0 +1,14 @@
+{
+    "CMCslug": "tangent",
```

Comment:
The address doesn't match what reported by coinmarketcap: https://coinmarketcap.com/currencies/tangent/
Explorer also shows that there are only 8 holders: https://cexplorer.io/asset/asset1pm727v4e3q6tlrclxh68cplv42mru67aepa3mc
Would you please check if this is correct? Thanks.